pkg/restapi/verifier/operation/operations_test.go

```diff
@@ -1653,7 +1694,8 @@ const (
 	prCardVC = `{
 	  "@context": [
 		"https://www.w3.org/2018/credentials/v1",
-		"https://w3id.org/citizenship/v1"
+		"https://w3id.org/citizenship/v1",
+        "https://w3id.org/vc-revocation-list-2020/v1"
```

Comment:
indentation looks off?

Reply:
I'm not sure why it's showing like this. looks good in goland